lab_3_keywords_textrank/main.py

```diff
@@ -152,7 +169,13 @@ def decode(self, encoded_tokens: tuple[int, ...]) -> Optional[tuple[str, ...]]:
                 sequence of string tokens
         In case of out-of-dictionary input data, None is returned
         """
-        pass
+        for index in encoded_tokens:
+            if index not in self._word2id:
+                return None
+        for index in encoded_tokens:
+            ind = self._id2word[index]
+        return tuple(ind)
```

Comment:
this way you are only going to return a tuple that consists of one element -- the last word in the encoded token sequence